SpatialGDK/Source/SpatialGDK/Public/SpatialConstants.cxx

```diff
@@ -0,0 +1,249 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#define LOCTEXT_NAMESPACE "SpatialConstants"
+
+using namespace SpatialGDK;
+
+namespace SpatialConstants
+{
+
+FString RPCTypeToString(ERPCType RPCType)
```

Comment:
Could potentially return const TCHAR* and let the caller deal with string initialization if FString was chosen as storage

Reply:
I tried this version to have it interact with FName, but apparently extracting a const TCHAR* from a FName is impossible, so I gave up. String is going to be the eventual container.

Reply:
Hmm logging could have been a good point, but in my RPC refactor, I'm moving away from RPCType to identify channels in favor of FName. So I'm likely never going to use these methods after initializing my structures, and logging from the stored FName of FString (I won't have raw TCHAR* around)